src/StoreKeeper/WooCommerce/B2C/Backoffice/MetaBoxes/ProductSyncMetaBox.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+namespace StoreKeeper\WooCommerce\B2C\Backoffice\MetaBoxes;
+
+use Exception;
+use StoreKeeper\WooCommerce\B2C\Commands\SyncWoocommerceSingleProduct;
+use StoreKeeper\WooCommerce\B2C\Commands\WebCommandRunner;
+use StoreKeeper\WooCommerce\B2C\I18N;
+use StoreKeeper\WooCommerce\B2C\Models\TaskModel;
+use StoreKeeper\WooCommerce\B2C\Options\StoreKeeperOptions;
+use StoreKeeper\WooCommerce\B2C\Tools\TaskHandler;
+
+class ProductSyncMetaBox extends AbstractMetaBox
+{
+    const ACTION_NAME = 'sk_sync_product';
+    const POST_TYPE = 'product';
+
+    final public function register(): void
+    {
+        $action = get_current_screen()->action;
+
+        if ('add' !== $action) {
+            add_meta_box(
+                'storekeeper-product-sync',
+                __('Product synchronization'),
+                [$this, 'renderSyncBox'],
+                self::POST_TYPE,
+                'side',
+                'high'
+            );
+        }
+    }
+
+    final public function renderSyncBox($post): void
+    {
+        $product = wc_get_product($post->ID);
+        $syncUrl = esc_attr($this->getNonceSyncActionLink($post));
+        $storekeeperId = $this->getPostMeta($product->get_id(), 'storekeeper_id', false);
```

Comment:
Map to int at this point

```suggestion
        $storekeeperId = (int) $this->getPostMeta($product->get_id(), 'storekeeper_id', false);
```